geo/src/algorithm/haversine_distance.rs

```diff
@@ -9,8 +9,13 @@ use num_traits::FromPrimitive;
 ///
 /// [haversine formula]: https://en.wikipedia.org/wiki/Haversine_formula
 ///
-/// *Note*: this implementation uses a mean earth radius of 6371.088 km, based on the [recommendation of
-/// the IUGG](ftp://athena.fsv.cvut.cz/ZFG/grs80-Moritz.pdf)
+/// *Note*: this implementation uses a mean earth radius of 6371.088 km, based on the recommendation of
```

Comment:
```suggestion
/// *Note*: this implementation uses a mean earth radius of 6371.0088 km, based on the recommendation of
```

Reply:
Come to think of it, I think this doc comment typo was the thing that originally sent me down this spiral of confusion.

Reply:
Yep this is once again also confusing me. Sigh.

Reply:
OK, now using `6371.0088` *and* the original value in metres from the citation (`6_371_008.7714 m`)